2nd-gen/packages/swc/components/progress-circle/stories/progress-circle.stories.ts

```diff
@@ -88,21 +99,65 @@ export const Playground: Story = {
 // ──────────────────────────
 
 /**
- * A progress circle consists of several key parts:
+ * ### Visual structure
+ *
+ * A progress circle consists of:
+ *
+ * 1. **Track** - Background ring showing the full progress range
+ * 2. **Fill** - Colored ring segment showing current progress
+ * 3. **Label** - Accessible text describing the operation (not visually rendered)
+ *
+ * ### Technical structure
+ *
+ * #### Slots
+ *
+ * - **Default slot**: Alternative way to provide an accessible label (the `label` attribute is preferred)
+ *
+ * #### Properties
+ *
+ * - **progress**: Numeric value from 0-100 controlling the fill amount
+ * - **label**: Accessible text describing what is loading or progressing
+ * - **indeterminate**: Boolean controlling animated vs. determinate state
+ * - **size**: Visual size (s, m, or l)
+ * - **static-color**: Color variant for use on different backgrounds (white or black)
+ *
+ * #### CSS custom properties
  *
- * - An accessible label (via `label` attribute)
- * - A progress value (via `progress` attribute)
- * - An indeterminate state (via `indeterminate` attribute)
- * - An optional size
- * - An optional static color for backgrounds that have color
+ * Key design tokens that control appearance:
+ *
+ * - **--spectrum-progress-circle-size**: Overall diameter of the circle
+ * - **--spectrum-progress-circle-thickness**: Width of the progress ring
+ * - **--spectrum-progress-circle-track-border-color**: Color of the background track
+ * - **--spectrum-progress-circle-fill-border-color**: Color of the progress fill
```

Comment:
In the past we've preferred not to document these because we don't recommend directly changing them